Jenkinsfile

```diff
@@ -910,13 +910,30 @@ def pullContainerImages(String mavenArgs) {
 	if ( containerImageRefs.length == 0 ) {
 		return
 	}
-	docker.withRegistry('https://index.docker.io/v1/', 'hibernateci.hub.docker.com') {
-		// Cannot use a foreach loop because then Jenkins wants to serialize the iterator,
-		// and obviously the iterator is not serializable.
```

Comment:
You might want to keep this comment at the top level, as the index-based loops are likely to raise an eyebrow or two.